src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/impl/FlowServiceImpl.java

```diff
@@ -170,17 +169,17 @@
         final String correlationId = RequestCorrelationId.getId();
         logger.info("Create flow: {}", input);
 
-        FlowCreateRequest payload;
+        FlowRequest flowRequest;
         try {
-            payload = new FlowCreateRequest(new FlowDto(input), input.getDiverseFlowId());
+            flowRequest =  flowMapper.toFlowCreateRequest(input);
```

Comment:
Are we ready for a complete switch to H&S create? I.e. remove toggle for a fast rollback on v1 create? (Current mapping have such toggle).

Reply:
It should be just tested, whole point to change worker on permanent basis